app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,20 @@
+class ReviewsController < ApplicationController
+
+  def new
+    @shelter_id = params[:shelter_id]
+  end
+
+  def create
+    review = Review.new(review_params)
+    review.shelter_id = params[:shelter_id]
+    review.save!
+    redirect_to "/shelters/#{params[:shelter_id]}"
+  end
```

Comment:
Confused, can we include the `:shelter_id` in the review_params below?

Reply:
You are absolutely correct. shelter_id can be included in the review_params so I will fix this as well.